framework/src/test/java/com/epam/ta/page/CloudPricingCalculatorPage.java

```diff
@@ -0,0 +1,182 @@
+    private WebElement numberOfGPUsButton;
+
+    @FindBy(id = "select_option_339")
+    private WebElement selectedNumberOfGPUs;
+
+    @FindBy(id = "select_value_label_333")
+    private WebElement GPUTypeButton;
+
+    @FindBy(id = "select_option_346")
+    private WebElement selectedGPUType;
+
+    @FindBy(id = "select_value_label_169")
+    private WebElement localSSDButton;
+
+    @FindBy(id = "select_option_233")
+    private WebElement selectedLocalSSD;
+
+    @FindBy(id = "select_value_label_56")
+    private WebElement datacenterLocationButton;
+
+    @FindBy(id = "select_option_181")
+    private WebElement selectedDatacenterLocation;
+
+    @FindBy(id = "select_value_label_57")
+    private WebElement commitedUsageButton;
+
+    @FindBy(id = "select_option_90")
+    private WebElement selectedCommitedUsage;
+
+    @FindBy(xpath = "//*[@aria-label='Add to Estimate']")
+    private WebElement addToEstimateButton;
+
+    @FindBy(id = "email_quote")
+    private WebElement emailEstimateButton;
+
+    @FindBy(xpath = "//input[@type='email']")
+    private WebElement emailPaste;
+
+    @FindBy(xpath = "//*[@aria-label='Send Email']")
+    private WebElement sendEmailButton;
+
+    @FindBy(xpath = "//*[@id='resultBlock']//h2/b")
+    private WebElement totalEstimatedCost;
+
+    public CloudPricingCalculatorPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public CloudPricingCalculatorPage switchFrame() {
+        driver.switchTo().frame(iframe).switchTo().frame(myFrame);
+        return this;
+    }
+
+    public CloudPricingCalculatorPage addNumberOfInstances(Engine engine) {
+        numberOfInstances.sendKeys(engine.getNumberOfInstances());
+        return this;
+    }
+
+    public CloudPricingCalculatorPage chooseOperatingSystem() {
+        operatingSystemButton.click();
+        clickAsJavascriptExecutor(selectedOperatingSystem);
+        return this;
+    }
+
+    public CloudPricingCalculatorPage chooseVMClass() {
+        clickAsJavascriptExecutor(vMClassButton);
+        clickAsJavascriptExecutor(selectedVMClass);
+        return this;
+    }
+
+    public CloudPricingCalculatorPage chooseInstanceType() {
+        clickAsJavascriptExecutor(instanceTypeButton);
+        clickAsJavascriptExecutor(selectedInstanceType);
+        return this;
+    }
+
+    public CloudPricingCalculatorPage chooseAddGPUs() {
+        clickAsJavascriptExecutor(addGPUsBox);
+        return this;
+    }
+
+    public CloudPricingCalculatorPage chooseNumberOfGPUs() {
+        clickAsJavascriptExecutor(numberOfGPUsButton);
+        clickAsJavascriptExecutor(selectedNumberOfGPUs);
+        return this;
+    }
+
+    public CloudPricingCalculatorPage chooseGPUType() {
+        clickAsJavascriptExecutor(GPUTypeButton);
+        clickAsJavascriptExecutor(selectedGPUType);
+        return this;
+    }
+
+    public CloudPricingCalculatorPage chooseLocalSSD() {
+        clickAsJavascriptExecutor(localSSDButton);
```

Comment:
why do you use JavascriptExecutor instead of Selenium click()?

Reply:
Click () method doesn't work in this case.